dbt/include/athena/macros/materializations/models/table/create_table_as.sql

```diff
@@ -1,5 +1,11 @@
 {% macro athena__create_table_as(temporary, relation, sql) -%}
-  {%- set external_location = config.get('external_location', default=none) -%}
+  {%- set split_model_path = model.path.split('/') -%}
+  {%- set domain_name = split_model_path[0] -%}
+  {%- set database_name = split_model_path[1] -%}
+  {%- set table_name = split_model_path[-1].split('.')[0] -%}
+  {%- set extraction_timestamp = run_started_at.strftime("%Y-%m-%d %H:%M:%S") -%}
+  {%- set default_external_location = 's3://mojap-derived-tables/domain=' + domain_name + '/database=' + database_name + '/table=' + table_name + '/extraction_timestamp=' + extraction_timestamp + '/' -%}
```

Comment:
I think as this is written in quite a few places in the repo it's okay to hardcode it here for now

However, could we break up this line? It's pretty hard to read as it is. Not sure if Jinja has a straightforward line breaking option, but if not it might work by adding lines to create two substrings then have this line join them together.

Reply:
Yes, I should have done that. 🤦‍♂️